src/types.ts

```diff
@@ -1,9 +1,156 @@
-export type Vote = {
-  index: number,
-  vote: boolean,
-  referendum: Referendum
-}
-
-export type Referendum = {
-  index: number,
-}
\ No newline at end of file
+import BigNumber from 'bignumber.js';

```

Comment:
polkadot.js apis is using [BN.js](https://github.com/indutny/bn.js/). Any reason not to use that instead?!

Reply:
Good catch, I mixed my bignumbers :D